src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -136,9 +136,15 @@ std::string postUpdateContext
 
 
     //
-    // 3. Render an XML-string of the request we want to forward
+    // 3. Render a string of the request we want to forward, forced to XML
+    //    FIXME P8: The format of this string (XML or JSON) should depend on the
+    //              format of the incoming message, but right now we force it to XML.
     //
-    std::string payloadIn = ucrP->render(ciP, UpdateContext, "");
+    ConnectionInfo ci;
+
+    ci.outFormat = XML;
+
+    std::string payloadIn = ucrP->render(&ci, UpdateContext, "");
```

Comment:
What about forwarded queries? It seems that the PR only solves the updateContext case...